src/azure-cli-core/azure/cli/core/auth/binary_cache.py

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-import collections.abc as collections
+import collections.abc
```

Comment:
It's not best practice to use the same name for an alias as a built-in module.
`import collections.abc as collections_abc` also works, but it adds more keystrokes since there is only one usage of `abc`.